config.default.php

```diff
@@ -89,7 +89,7 @@
 	'limits' => array(
 
 		# Duration in seconds of the login cookie.
-		'cookie_duration' => 2592000,
+		'cookie_duration' => COOKIE_DURATION,
```

Comment:
@Alkarex What about this one?

Reply:
`FreshRSS_FormAuth::DEFAULT_COOKIE_DURATION` should work.
Was that your question?

Reply:
No, more that it comes across weirder/more confusing to me.

Reply:
Then maybe leaving it as an integer there would be simpler

Reply:
I wasn't objecting (much), but also it doesn't seem to work.
```
[Sat Dec 26 21:39:27.542015 2020] [php7:error] [pid 19] [client 127.0.0.1:49252] PHP Fatal error:  Uncaught Error: Class 'FreshRSS_FormAuth' not found in /var/www/FreshRSS/config.default.php:92
```

Reply:
Ah sorry, should have been in `FreshRSS_Auth::*`

Reply:
Another option being to extract the `FreshRSS_FormAuth` class in its own PHP file

Reply:
Drat, I did that first before your other comment but I never committed it.